setup.py

```diff
@@ -2,13 +2,13 @@
 
 
 setup(
-    name='django-serial-forms',
-    py_modules=['serial_forms'],
+    name='django-adapters',
+    py_modules=['adapters'],
     version='0.1',
-    description='Constructible, introspectable serializers and forms',
+    description='Constructible, introspectable adaption for serialization and forms',
```

Comment:
For me "adaption" sounds quite different than what "adapter" would mean, maybe change it to "adapters"?